### DISCUSSION TITLE
Error when installing numpy

### DISCUSSION THREAD

**Discussion by batmanscode** [community/community#32049] [Codespaces]
I get this error when using https://github.com/batmanscode/pydacc on codespaces.

Pycaret is in my requirements and is what's installing numpy. What can I do to get this to install?

This seems like a problem with the environment since this installs as expected when deployed on https://railway.app.

```
      × Encountered error while trying to install package.
      ╰─> numpy
      
      note: This is an issue with the package mentioned above, not pip.
      hint: See above for output from the failure.
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

× pip subprocess to install build dependencies did not run successfully.
│ exit code: 1
╰─> See above for output.
```

**Comment by reybard** [comment]
The environment is configurable to the codespace. I am not sure from the provided output anything about that environment. What OS base image are you running on? What version of Python do you have installed? How did you install it (brew, package manager, etc)? Can you install other modules? Is there more detailed debug output we could see to know precisely what about the installation of numpy failed? A codespace should not prohibit the installation of python or any of its popular modules by default.

**Reply by batmanscode** [reply]
Thank you for responding. This is running on the recommended default python 3.10. The devcontainer was created using the add devcontainer option once VS code was launched. The package manager is pip. And yes it seems the other packages have been installed!

If you have the time please do try to launch the repo in codespaces, it's set to install the packages automatically
